### PR TITLE
Fix of base url

### PR DESCRIPTION
**Pull request by Khukhuna**
Changes:
- Fix of base url (I accidently pushed my heroku deploy in one of my pr)
- Added AuthGuard in providers

### CODE REVIEW

File: src/app/environments/environments.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 export class environment {
     public static production = false;
-    public static baseUrl = 'https://jbossnewserver.herokuapp.com/';
+    public static baseUrl = 'https://cert-gen-jboss.herokuapp.com/';
```
Comment (aashutoshrathi) [inline src/app/environments/environments.ts]: Have you checked this one? Is there working perfectly?

Comment (nikitadhiman) [reply src/app/environments/environments.ts]: Exactly my concern, rest everything seems fine to me too

Comment (aashutoshrathi) [reply src/app/environments/environments.ts]: Last, I checked we got `500: Server Error` on user creation

Comment (Khukhuna) [reply src/app/environments/environments.ts]: ``` https://jbossnewserver.herokuapp.com/ ``` was my deploy on heroku, which I deleted.

So I changed it to:
``` https://cert-gen-jboss.herokuapp.com/ ``` (official server url, given in [cert-gen-server repository](https://github.com/JBossOutreach/certificate-generator-server))

That 500: Serer error should be a bug in [certificate-generator-server](https://github.com/JBossOutreach/certificate-generator-server) repository.

Comment (aashutoshrathi) [reply src/app/environments/environments.ts]: Yeah, there is some issue with the deployment that we have to check anyway.
Moving to the official URL is right, I think :+1:

**Pull request merged**